modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-cluster-tasks-ref_{context}"]
+= Tasks provided in the {pipelines-shortname} namespace
+
+A {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortnames} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
+
+`ClusterTask` functionality is deprecated since version 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
```

Comment:
Might be obvious, but wondering if you should say specify which version

```suggestion
`ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
```